src/core/variable.jl

```diff
@@ -612,14 +527,14 @@ function variable_mc_transformer_flow_reactive(pm::_PMs.AbstractPowerModel; nw::
     )
 
     if bounded
-        for arc in _PMs.ref(pm, nw, :arcs_trans)
-            tr_id = arc[1]
-            flow_lb  = -_PMs.ref(pm, nw, :transformer, tr_id, "rate_a")
-            flow_ub  =  _PMs.ref(pm, nw, :transformer, tr_id, "rate_a")
-            for c in cnds
-                JuMP.set_lower_bound(qt[arc][c], flow_lb[c])
-                JuMP.set_upper_bound(qt[arc][c], flow_ub[c])
-            end
+        for arc in _PMs.ref(pm, nw, :arcs_from_trans)
+            (t,i,j) = arc
+            rate_a_fr, rate_a_to = _calc_transformer_power_ub_frto(_PMs.ref(pm, nw, :transformer, t), _PMs.ref(pm, nw, :bus, i), _PMs.ref(pm, nw, :bus, j))
+
+            JuMP.set_lower_bound.(qt[(t,i,j)], -rate_a_fr)
+            JuMP.set_upper_bound.(qt[(t,i,j)],  rate_a_fr)
+            JuMP.set_lower_bound.(qt[(t,j,i)], -rate_a_fr)
+            JuMP.set_upper_bound.(qt[(t,j,i)],  rate_a_fr)
```

Comment:
should we instead be using the internal `set_lower_bound` and `set_upper_bound` here?